BLSQ/iaso
# IA-1808: Links - Replace HOC with hooks

`LinksCompare` and `LinksValue` still used high order components to get the styling and translations. 

## Self proof reading checklist

## Changes

- removed `withStyles` and `injectIntl` from components
- renamed `LinksValue`'s `renderValue` as `LinkValue` and spun it into its own file
- removed unused variable that looked suspicious (but was unused)
- Convert the lot to TS


## How to test

Go to Org Units>Details>Links: nothing should change
Go to Links Validation: nothing should change

## Print screen / video

No visible changes
